README.md

```diff
@@ -1,22 +1,628 @@
+    - list
+    - watch
+    - update
+    - patch
+    - delete
+EOF
+```
+<br/>
+<br/>
+
+Please run the following command
+
+```
+kubectl apply -f - <<EOF
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: argo-workflow-binding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: argo-workflow-role
+subjects:
+- kind: ServiceAccount
+  name: argo
+  namespace: argo
+EOF
+```
+<br/>
+<br/>
+
+g. Run the following command. The output should say 'yes'.
+
+`kubectl auth can-i create workflowtaskresults.argoproj.io -n argo --as=system:serviceaccount:argo:argo`
+
+<br/>
+<br/>
+
+h. Optional: check roles and role-bindings
+
+`kubectl get role -n argo`
+
+<img width="660" alt="Screenshot 2025-06-20 at 10 27 29 AM" src="https://github.com/user-attachments/assets/a62b6253-1c89-4008-bbbb-8c8403f5db45" />
+
+<br/>
+<br/>
+
+`kubectl describe role <role_name> -n argo`
+
+<img width="807" alt="Screenshot 2025-06-20 at 10 29 35 AM" src="https://github.com/user-attachments/assets/0357d932-ca01-4e52-8762-c8480700e3ed" />
+
+<img width="822" alt="Screenshot 2025-06-20 at 10 30 17 AM" src="https://github.com/user-attachments/assets/9ab3d32a-4239-48d0-8129-1a191b0b2a76" />
+
+<br/>
+<br/>
+
+### 7. Clone ofo-argo repository to Master instance
+
+In the home directory of your terminal, type in the following
+
+`git clone https://github.com/open-forest-observatory/ofo-argo.git`
+
+NOTE: if you want to use a development branch of the repo. eg, `git checkout docs/JG/readme-editing`
+
+<br/>
+
+### 8. Connect VM instances to shared volume
+
+The following is about connecting the VM instances with the `/ofo-share` volume so it can read the drone imagery to process and write the outputs.
+
+a. Install Network File System (NFS) on all instances including the master and each worker. You will need to connect to each VM terminal to do installation. 
+
+`sudo apt update`
+
+`sudo apt install nfs-common -y`
+
+
+<br/>
+
+b. Reconnect to the master instance and navigate into the cloned repository
+
+`cd ~/ofo-argo`
+
+<br/>
+
+c. Set up the persistent volumes (PV) defined by the workflow. You are specifying the read (raw drone imagery on `ofo-share`) and the write (metashape output location)
+
+`kubectl apply -f argo-output-pv.yaml`
+
+`kubectl apply -f ofo-share-pv.yaml`
+
+<br/>
+
+d. Set up persistent volume claims (PVC)
+
+`kubectl apply -f argo-output-pvc.yaml -n argo`
+
+`kubectl apply -f ofo-share-pvc.yaml -n argo`
+
+<br/>
+
+e. Check the PVs
+
+`kubectl get pv`
+
+`kubectl get pvc -n argo`
+
+<br/>
+<br/>
+<br/>
+
+## Run the Workflow
+
+### 1. Declare the ip address of the metashape license server
+
+On the master instance terminal, type:
+
+`export AGISOFT_FLS=<ip_address>:5842`
+
+This variable will only last during the terminal session and will have to be re-declared each time you start a new master terminal. We are not putting the ip address here to prevent unauthorized people from using it. 
```

Comment:
One option would be to link the credentials docs, since it's restricted, but maybe we'd get spam request to share it.